common/src/main/scala/explore/components/state/ConnectionManager.scala

```diff
@@ -29,6 +30,10 @@ object ConnectionManager {
 
   protected case class State(initialized: Boolean = false)
 
+  protected implicit val propsReuse: Reusability[Props] =
+    Reusability.derive && Reusability.by(_.render)
```

Comment:
Is this because `derive` only takes the first set of params?

Reply:
Yup

Reply:
Heh. Good question, Carlos.